fixture/test_dataservice_mongohostdata_02.json

```diff
@@ -0,0 +1,432 @@
+{
+        "ID": "5dd41aa2b2fa40163c878538",
```

Comment:
It has the same id as `fixture/test_dataservice_mongohostdata_02.json`

Reply:
This IS fixture/test_dataservice_mongohostdata_02.json !

Reply:
(I understood)